esp/src/eclwatch/WUDetailsWidget.js

```diff
@@ -156,16 +156,21 @@ define([
 
             _onCancelDialog: function () {
                 this.zapDialog.hide();
-                this.emailCheckbox.set("checked", false);
                 this.emailCheckbox.set("value", "false");
```

Comment:
While not a change in this PR - setting the value of the emailCheckbox to "false" is probably wrong here?